data/json/monstergroups.json

```diff
@@ -292,6 +292,7 @@
       { "monster" : "mon_worm", "freq" :  1, "cost_multiplier" : 0, "starts" : 336, "conditions" : ["SPRING", "SUMMER", "AUTUMN"] },
       { "monster" : "mon_worm", "freq" :  1, "cost_multiplier" : 0, "starts" : 504, "conditions" : ["SPRING", "SUMMER", "AUTUMN"] },
       { "monster" : "mon_worm", "freq" :  1, "cost_multiplier" : 0, "starts" : 672, "conditions" : ["SPRING", "SUMMER", "AUTUMN"] },
+	  { "monster" : "mon_nakedmolerat_giant", "freq" : 1, "cost_multiplier" : 3,                 "conditions" : ["DUSK", "NIGHT"] },
```

Comment:
Tabs

Reply:
Everything should be properly linted, what's missing exactly?

Reply:
The linter may be unable to change tabs into spaces. You may need to do it manually. Though if it doesn't change tabs to spaces, it may also be worth reporting as a bug.
If you're using notepad++, enable view->show symbol->show white space and tab. Then search all the changed files for tabs, by pressing CTRL+F, setting "extended" search option, then searching for `\t`